docs/tokens.md

```diff
@@ -0,0 +1,257 @@
+token with a liquid market price.
+
+### **Supported tokens**
+
+RIF Rollup can be used to transfer RBTC and ERC20 tokens. The full list of currently supported tokens is available under
+this [link](https://explorer.rollup.rifcomputing.net) (opens new window).
+
+Depending on the type of transaction, the submitter may or may not be able to select the type of token to pay the fees
+with:
+
+Operations where the user can select any of the listed tokens suitable for payment of fees (liquid market price):
+
+- ChangePubKey
+- MintNFT
+- WithdrawNFT
+- Swap
+
+Operations where the user must use the same token being operated as the fees token:
+
+- Withdraw
+- ForcedExit
+- Transfer
+
+> DISCLAIMER: Listing of a token on RIF Rollup does not represent an endorsement for this token by the RIF Rollup team,
+> IOVLabs, or any other entity. We do not have competence and authority to analyze the token business model and smart
+> contract security guarantees of a particular token. Please use any token on your own risk and judgement.
+
+### **Tokens not available for fee payment**
+
+In some operations (Withdraw, ForcedExit, Transfer) the fee token cannot be selected and must be paid using the token
+being transferred or withdrawn. However, if the transaction involves a token without a liquid market price, then (as per
+RIF Rollup documentation) fees for these operations are supposed to be paid using a different token but unfortunately,
+this is currently not implemented.
+
+Trying to transfer or withdraw a token that doesn’t have a liquid market price will **fail**. There is a
+[workaround](https://discord.com/channels/722409280497516566/829104124145696808/920231908460335114) though. To pay the
+fee in a different token, you need to create a batch of transactions (there is a convenience class for it in the
+[JS SDK](https://docs.zksync.io/api/sdk/js/accounts/#batch-builder)). The first transaction should be of the token you
+wish to transfer/withdraw. Then since we’re assuming a scenario where no other tx needs to be sent, the user must
+include a dummy transaction whose sole purpose is to pay for the fees. This second transaction may be of the same type
+(i.e. transfer/withdraw) but for one of the available token fees. Since it’s a dummy transaction the user will set the
+amount (transferred or withdrawn) as 0, and she must include a fee large enough (in this case approximately twice the
+size) to cover the cost for both transactions. While this method works, the
+[Transaction Batching](#transaction-batching) section presents a more user-friendly approach.
+
+### **Listing a token**
+
+1. Deploy your ERC20 contract and take note of its _address_
+
+   1. You can use `zk deploy-erc20 new <name> <symbol> <decimals> [<implementation>]` which will output
+
+      ```json
+      {
+        "address": "0x186E485150dDAB799Be8e25f87D717dEc45a86e5",
+        "name": "ProRif",
+        "symbol": "PRIF",
+        "decimals": "18"
+      }
+      ```
```

Comment:
I tried to fix/research this, but to no avail. It seems like github markdown has an issue with json format? Not sure why it should, tho. If you see what the problem is, do let know.

Reply:
nw, it looks like it's an issue just during the PR, see [the branch](https://github.com/rsksmart/rif-rollup/blob/ROLLUP-197/erc20docs/docs/tokens.md#listing-a-token).